src/aura/BGE_EntryForm/BGE_EntryFormHelper.js

```diff
@@ -28,9 +43,70 @@
             }
         }
 
+        // assign opportunity/payment lookup and import status
+        const selectedDonation = component.get('v.selectedDonation');
+        const userSelectedMatch = $A.get('$Label.c.bdiMatchedByUser');
+        const userSelectedNewOpp = $A.get('$Label.c.bdiMatchedByUserNewOpp');
+        const applyNewPayment = $A.get('$Label.c.bdiMatchedApplyNewPayment');
+
+        if (selectedDonation) {
+            if (selectedDonation.attributes.type === 'Opportunity') {
+                //matched opportunity; create new payment or update opportunity
+                rowFields[labels.opportunityImportedLookupField] = selectedDonation.Id;
+                if (selectedDonation.applyPayment) {
```

Comment:
@bethbrains Beth is suggesting to take the reference to applied payment out until that story is being delivered